test/unit/specs/__snapshots__/transactions.test.ts.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Transactions Merge Transactions 1`] = `
```

Comment:
Snapshot Testing. The test should always generate the exact output. If not, it will show the diff.

It is my preferred way to validate complicated objects.